blog/what-is-slice.mdx

```diff
@@ -0,0 +1,45 @@
+
+Each network slice is logically separated from the other, is an independent part of the network, and targets different services with different requirements at the same time. This enables to build and manage a network that has the capabilities to meet the emerging requirements from a wide range of users and enable multi-tenancy in a multi-cluster Kubernetes setup.
+
+### Concept of a Slice in KubeSlice
+
+In KubeSlice, a Slice is a Kubernetes [custom resource](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) defined to implement multi-tenancy in a cluster. 
+[Multi-tenancy](https://kubernetes.io/docs/concepts/security/multi-tenancy/) is when a single instance of an application or a single Kubernetes cluster can serve multiple users, called tenants, at the same time without the problems of noisy/nosey neighbor, policy and configuration drift, and resource hogging creeping in.
+
+KubeSlice creates logical application **slice boundaries** in a cluster that allows pods and services to communicate efficiently within a cluster, across clusters on the cloud or datacenters. In order to communicate within a multi-cluster architecture, an overlay network (an application slice) following the RFC1918 protocol is created by KubeSlice that provides seamless connectivity between the pods of an application running in multiple clusters. Existing intra-cluster communication remains local to the cluster utilizing each pod’s CNI interface. 
+Kubeslice provides isolation of network traffic between clusters by creating an overlay network for inter-cluster communication.
+
+There are a lot of advantages achieved with this way of implementing multi-cluster multi-tenancy in Kubernetes:
+
+- **Better Resiliency** - Resiliency is the ability of an application to react to problems in one of its components and still provide the best possible service. Due to the overlay network created by Kubeslice, the application is separated into virtual clusters. The services in any virtual cluster can independently failover without disrupting services in other layers/clusters. These failovers can be done while intra-cluster communication continues at a reduced capacity. Therefore, connections can be re-routed to other clusters in a Slice, which ensures less disruptions and quick recoveries.
+By deploying an application across clusters and connecting using this slice overlay network, application services are able to achieve resiliency.
+To know more about resiliency in Kubeslice, refer to the [documentation](https://avesha.io/use-case/use-case-resiliency/)
+
+- **Traffic Control** - The ability to control traffic across nodes and clusters. The traffic control policy is maintained for all application pods on that slice. The slices in a cluster have a QoS profile, which is defined in each slice allowing granular traffic control between clusters.
+To know more about how do we achieve traffic control, refer to the [official documentation](#).
```

Comment:
```suggestion
To know more about how do we achieve traffic control, refer to the [official documentation](#).
```

Could you pls point me to the right page from the documentation, for this particular point?! 
Wasn't able to find one

Reply:
See [comment #1](https://github.com/kubeslice/docs/pull/70#discussion_r989829767) & [comment #2](https://github.com/kubeslice/docs/pull/70#discussion_r989830088) below.